pkg/operator/controller/ingress/deployment.go

```diff
@@ -325,7 +325,34 @@ func desiredRouterDeployment(ci *operatorv1.IngressController, ingressController
 	secretName := controller.RouterEffectiveDefaultCertificateSecretName(ci, deployment.Namespace)
 	deployment.Spec.Template.Spec.Volumes[0].Secret.SecretName = secretName.Name
 
+	var (
+		useSyslogSidecar  bool
+		useSyslogEndpoint bool
+		accessLogLevel    string
+		accessLogFormat   string
+		syslogEndpoint    string
+		syslogFacility    string
```

Comment:
format/endpoint/facility seem like extracted fields from SyslogLoggingDestinationParameters — what if you use the actual struct instead?

Reply:
Done (using the `AccessLogging` type).